src/plugins/plugin-https.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// Force load http to patch https with the http plugin.
+// https depends on http anyway, so this shouldn't cause an unnecessary load.
+require('http');
+
+module.exports = [];
```

Comment:
It doesn't make sense to use both CJS style exports and ES6 exports in the same file. Which one takes precedence?

Reply:
Changing this to `export = []`.